scripts/bash_utils.sh

```diff
@@ -40,3 +23,16 @@ function setup_python {
   fi
   echo ${python_bin}
 }
+
+function install_py_packages {
+  local deps="black==$BLACK_VERSION flake8==$FLAKE8_VERSION clang-format==$CLANG_FORMAT_VERSION clang-tidy==$CLANG_TIDY_VERSION"
```

Comment:
I'm kinda thinking we should force people to use a virtual environment.. currently we're not even warning that we're installing into the system python, we just do it via env var. I bet we're not the only c-ish project that uses the python wrappers. but just a suggestion, it's a bit paternal. on the other hand, nothing says what's gonna happen internally when `format.sh` is being run and I personally at least would like to have it printed as warning. I thought about actually asking for user input to get the chance to activate a virtual env before installing anything.

Reply:
I think that would be a good idea. As someone brand new to poking around with the sources of Valhalla it's not obvious how it all works until you pushed a few buttons to see what happens. Would be nice to avoid having a bunch of stuff installed to system.